pycytominer/cyto_utils/cp_aggregate.py

```diff
@@ -0,0 +1,138 @@
+"""
+Functions for aggregating "non-compartment" features
+"""
+
+import pandas as pd
+import numpy as np
+
+
+def aggregate_fields_count(image_df, strata, fields_of_view_feature):
+    """Compute the number of fields per well and create a new column called Metadata_Site_Count
+
+    Parameters
+    ----------
+    image_df : pandas.core.frame.DataFrame
+        Image table dataframe which includes the strata and fields of view feature as columns.
+    strata :  list of str
+        The columns to groupby and aggregate single cells.
+    fields_of_view_feature: str
+        Name of the fields of the view column.
+
+    Returns
+    -------
+    fields_count_df: pandas.core.frame.DataFrame
+        DataFrame with the Metadata_Site_Count column.
+
+    """
+
+    fields_count_df = image_df.loc[:, list(np.union1d(strata, fields_of_view_feature))]
+
+    fields_count_df = (
+        fields_count_df.groupby(strata)[fields_of_view_feature]
+        .count()
+        .reset_index()
+        .rename(columns={f"{fields_of_view_feature}": f"Metadata_Site_Count"})
+    )
+
+    return fields_count_df
+
+
+def aggregate_image_count_features(df, image_features_df, image_cols, strata):
+    """Aggregate the Count features in the Image table.
```

Comment:
what are the "Count features"? Is this just cell count?

Reply:
Yup. There is also cytoplasm and nucleus count